src/commands/mod.rs

```diff
@@ -1841,6 +1844,57 @@ fn edit_description(
     Ok(text_util::complete_newline(description))
 }
 
+fn edit_sparse(
```

Comment:
Note: I'm not entirely satisfied with this, since it's fairly similar to `edit_description()` above. A lot of the code is shared, but making it generic would introduce more parameters (filename, contents, and error messages differ; ideally we wouldn't pass the contents of the file directly to avoid useless temporary strings).

I can do this in this PR, or in a follow-up PR, or not at all. I also put `edit_sparse()` right below `edit_description()` since they share a lot of code, but now `edit_sparse()` is pretty far from its usage.

Reply:
That seems fine for now. Maybe it'll be clearer how to structure it once we have other similar functions.